docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
<p>Neither of these links seem to work (... /docs/source/technical_tutorials/docs/source/technical_tutorials/search/granules.ipynb is what the links are doing). Might could also directly link the pages in the Docs site</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
relative path is probably `../search/notebookname.ipynb`

Reply:
oopsies, meant to update these